### PR TITLE
Start inter-bot project

### PR DESCRIPTION
**Pull request by ChrisChinchilla**
@amuramoto Here's the inter-bot start

### CODE REVIEW

**Comment by amuramoto** [comment]
Readme refers to completed quick start code. Should refer to complete inter-bot code.

**Comment by ChrisChinchilla** [comment]
Again, can't completely merge yet until we have live tutorial link, but updated @amuramoto

File: inter-page/marketing-bot/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Inter-bot Tutorial - Starter Project for Marketing Bot
+
+This is a starter project you can use to as a starting point for completing the marketing bot of the [Messenger Platform inter-bot tutorial](#).
```
Comment (amuramoto) [inline inter-page/marketing-bot/README.md]: add link

File: inter-page/marketing-bot/app.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+ */
+
+'use strict';
+
+// Imports dependencies and set up http server
+const
+    request = require('request'),
+    express = require('express'),
+    body_parser = require('body-parser'),
+    ACCESS_TOKEN = process.env.ACCESS_TOKEN,
```
Comment (amuramoto) [inline inter-page/marketing-bot/app.js]: standardize. Either capitalize your constants or don't

File: inter-page/marketing-bot/app.js [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+        res.status(200).send('EVENT_RECEIVED');
+
+    } else {
+        // Return a '404 Not Found' if event is not from a page subscription
+        res.sendStatus(404);
+    }
+
+});
+
+function processComments(comment) {
```
Comment (amuramoto) [inline inter-page/marketing-bot/app.js]: add comment describing stub

File: inter-page/support-bot/README.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Inter-bot Tutorial - Starter Project for Support Bot
+
+This is a starter project you can use to as a starting point for completing the support bot of the [Messenger Platform inter-bot tutorial](#).
```
Comment (amuramoto) [inline inter-page/support-bot/README.md]: update link

File: inter-page/support-bot/app.js [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+        response = {
+            "text": `Sorry, I don't understand what you mean.`
+        }
+    }
+
+    // Send the response message
+    callSendAPI(sender_psid, response);
+}
+
+function handleIncoming(sender_psid, received_postback) {
```
Comment (amuramoto) [inline inter-page/support-bot/app.js]: add comment describing stub

File: inter-page/marketing-bot/app.js (hunk 2 above)
Comment (ChrisChinchilla) [reply inter-page/marketing-bot/app.js]: This has been the pattern in other examples and posts, but I will standardise for now.

**Comment by ChrisChinchilla** [comment]
@amuramoto Done

File: inter-page/marketing-bot/app.js (hunk 3 above)
Comment (amuramoto) [reply inter-page/marketing-bot/app.js]: @ChrisChinchilla  Please add a stub

File: inter-page/support-bot/app.js (hunk 5 above)
Comment (amuramoto) [reply inter-page/support-bot/app.js]: @ChrisChinchilla  description stub needed

**Comment by ChrisChinchilla** [comment]
@amuramoto Descriptions added

File: inter-page/marketing-bot/app.js [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+app.listen(process.env.PORT || 5000, () => console.log('webhook is listening'));
+
+// Accepts POST requests at the /webhook endpoint
+app.post('/webhook', (req, res) => {
+    // Parse the request body from the POST
+    let body = req.body;
+
+    // Check the webhook event is from a Page subscription
+    if (body.object === 'page') {
+        body.entry.forEach(function (entry) {
```
Comment (amuramoto) [inline inter-page/marketing-bot/app.js]: @ChrisChinchilla convert to arrow function

**Pull request merged**